src/sass/table/table.scss

```diff
@@ -164,6 +168,32 @@ $density-relaxed-img-radius: 8px;
     padding-inline-end: 0;
 }
 
+.table--loading-state {
+    overflow: hidden;
+}
+
+.table--loading-state table {
+    opacity: 0.15;
+    user-select: none;
+}
+
+.table--loading-state img,
+.table--loading-state .signal {
+    filter: var(--color-media-disabled-filter);
+}
+
+.table--loading-state a {
+    cursor: default;
+    pointer-events: none;
+    text-decoration: none;
+}
+
+.table--loading-state .progress-bar-expressive {
+    bottom: calc(50%);
+    left: calc(50% - 172px);
```

Comment:
I noticed that its _slightly_ offcenter and overflowing out of the container
<img width="760" alt="Screenshot 2024-11-04 at 8 09 29 AM" src="https://github.com/user-attachments/assets/32115166-7b7a-490d-aa32-bf2c721c0581">

If I put left: 0 and right: 0, it seems to center it fine.

Reply:
@agliga , great catch! Will do.